docs/hub/security.md

```diff
@@ -15,6 +15,7 @@ For any other security questions, please feel free to send us an email at securi
 - [User Access Tokens](./security-tokens)
 - [Git over SSH](./security-git-ssh)
 - [Signing commits with GPG](./security-gpg)
+- [Single Sign-On (SSO)](./security-sso)
```

Comment:
I'm wondering if it wouldn't make more sense under `Organizations`, but no strong opinion, cc @julien-c

Reply:
i would keep it under security but link it from https://huggingface.co/docs/hub/organizations , https://huggingface.co/docs/hub/organizations-managing and from https://huggingface.co/docs/hub/organizations-security

Reply:
I'll merge without it. Let's do that when I have the UI ready